log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/VizGrammar.js

```diff
@@ -0,0 +1,2300 @@
+            "fillOpacity": {"value": 1}
+        };
+    } else {
+        markContent = {
+            "x": {"scale": "x", "field": metadata.names[config.x]},
+            "width": {"scale": "x", "band": true, "offset": -1},
+            "y": {"scale": "y", "field": metadata.names[config.y]},
+            "y2": {"scale": "y", "value": 0},
+            "fill": {"value": config.markColor},
+            "fillOpacity": {"value": 1}
+        };
+    }
+
+    var mark = {
+        "type": "rect",
+        "from": {"data": config.title},
+        "properties": {
+            "update": markContent,
+            "hover": {
+                "fillOpacity": {"value": 0.5}
+            }
+        }
+    };
+
+
+    return mark;
+}
+
+function getStackBarMark(config, metadata){
+
+    var markContent;
+    if (config.orientation == "left") {
+        mark = {
+            "type": "rect",
+            "from": {
+                "data": config.title,
+                "transform": [
+                    { "type": "stack",
+                        "groupby": [metadata.names[config.x]],
+                        "sortby": [metadata.names[config.color]],
+                        "field":metadata.names[config.y]}
+                ]
+            },
+            "properties": {
+                "update": {
+                    "y": {"scale": "x", "field": metadata.names[config.x]},
+                    "height": {"scale": "x", "band": true, "offset": -1},
+                    "x": {"scale": "y", "field": "layout_start"},
+                    "x2": {"scale": "y", "field": "layout_end"},
+                    "fill": {"scale": "color", "field": metadata.names[config.color]},
+                    "fillOpacity": {"value": 1}
+                },
+                "hover": {
+                    "fillOpacity": {"value": 0.5}
+                }
+            }
+        };
+    } else {
+        mark = {
+            "type": "rect",
+            "from": {
+                "data": config.title,
+                "transform": [
+                    { "type": "stack",
+                        "groupby": [metadata.names[config.x]],
+                        "sortby": [metadata.names[config.color]],
+                        "field":metadata.names[config.y]}
+                ]
+            },
+            "properties": {
+                "update": {
+                    "x": {"scale": "x", "field": metadata.names[config.x]},
+                    "width": {"scale": "x", "band": true, "offset": -1},
+                    "y": {"scale": "y", "field": "layout_start"},
+                    "y2": {"scale": "y", "field": "layout_end"},
+                    "fill": {"scale": "color", "field": metadata.names[config.color]},
+                    "fillOpacity": {"value": 1}
+                },
+                "hover": {
+                    "fillOpacity": {"value": 0.5}
+                }
+            }
+        };
+    }
+
+
+
+
+    return mark;
+}
+
+function getGroupBarMark(config, metadata){
```

Comment:
Hi Chandana,
We have replace the Vizgrammer.js with Vizgrammer.min.js and relevant commits are in

https://github.com/wso2/shared-analytics/pull/126/files#diff-6ac780d61b2b6d470c8ff04bdeed3bcdR52

Thanks,
SajithD